Prose/ProseLib/Sources/AuthenticationFeature/BasicAuthReducer.swift

```diff
@@ -125,18 +114,14 @@ public let basicAuthReducer = Reducer<
         state.focusedField = .password
 
     case .submitTapped(.password):
-        if state.isLogInButtonEnabled {
-            return Effect(value: .logIn)
-        }
```

Comment:
I'm not fan of removing this 😕 Yes it was a bit redeundant as the form is disabled when loading, but at least here it's clear that we will never trigger the login twice.

Reply:
Same as above. Follow the program flow, the logic hasn't changed.

Reply:
Yes, I had looked for it. But same as above, it's just a feeling. Let's leave it like this, I was just commenting.